tests/geobase/CMakeLists.txt

```
@@ -15,3 +15,16 @@ add_library(${checks_lib} INTERFACE)
 target_link_libraries(${checks_lib} INTERFACE FairRoot::GeoBase Catch2::Catch2 fmt::fmt)
 target_include_directories(${checks_lib} INTERFACE ${CMAKE_SOURCE_DIR})
 fairroot_library_settings(${checks_lib})
+
+fairroot_add_catch2_test_suite(${suite}_FairGeoSet
```

```suggestion
fairroot_add_catch2_test_suite(${suite}
```

Let's start with one suite per FairRoot target.

---

Done.